electron/config.ts

```diff
@@ -243,8 +244,8 @@ abstract class GlobalConfig {
     // Known places where Steam might be found.
     // Just add a new string here in case another path is found on another distro.
     const steamPaths = [
-      join(homedir(), '.steam'),
-      join(homedir(), '.var/app/com.valvesoftware.Steam/.local/share/Steam'),
+      join(home, '.steam'),
```

Comment:
Here we should also use `homedir()` because on Flatpak it won't find Steam proton, especially on the Deck.
Thats why perhaps we can kill the `home` variable, or change it to be `homedir` and if we need something specific for the flatpak home we can use `flatpakHome`.

Reply:
Alright, I'll get on that once I'm home